tests/test.hs

```diff
@@ -779,6 +779,7 @@ textIgnored :: [FilePath]
 textIgnored 
   = [ "Setup.lhs"
     -- , "Data/Text/Axioms.hs"
+    , "Data/Text/Foreign.hs"                -- Breaks on GHC 9.0.1
```

Comment:
IIUC this disables the test in all configurations, not just GHC 9.0.1.

There is this PR that disabled this test in the CI configuration:
https://github.com/ucsd-progsys/liquidhaskell/pull/1872

I guess the options are:
1. Use CPP macros skip tests for ghc9, or
2. Update the CI configuration after the changes to the test name

In general, I prefer to avoid CPP, but you let me know if the current arrangement is too confusing.

Reply:
Because 1) `test.hs` already makes use of CPP; 2) the fact there are two ignore lists for tests is highly confusing (to me, at least); and 3) the name of the test in question is generated and depends on the number of tests run before it, and so cannot be reliably managed outside of `test.hs`; I propose that CPP is the preferable answer in this case.

Reply:
Works for me.